### DISCUSSION TITLE
Welcome to RiMusic Discussions!

### DISCUSSION THREAD

**Discussion by fast4x** [fast4x/RiMusic#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fast4x** [comment]
I am preparing the release of version 0.6.16.1 as an intermediate version for publication in F-Droid.

**Reply by ikanakova** [reply]
I suppose existing requests will be merged after the new version is released?
I also noticed that the README states "Flaticon App icon based on icon created by Ilham Fitrotul Hayat.", which is no longer accurate as it refers to the ViMusic icon. But I'll fix it after the release of 0.6.16.1. And I also have the fastlane metadata translation ready.

**Reply by fast4x** [reply]
I would merge everything that is ready now.

**Reply by fast4x** [reply]
All merged, i prepare the release

**Comment by Sahl1** [comment]
I didn't quite understand the medley duration feature. Please if any one of you could explain it to me

**Reply by fast4x** [reply]
If you set a duration, songs are played dor this time and after it auto switch to next ...

**Reply by aneesh1122** [reply]
> If you set a duration, songs are played dor this time and after it auto switch to next ...

Isn't it supposed to switch only when there are errors in loading the song?

**Reply by fast4x** [reply]
If you activate the Medley the song plays for the number of seconds set, then skip to the next one,bid error occur never change song now.

**Comment by fast4x** [comment]
Yesterday it was not possible to publish the new version,  I should be able by tonight.

**Comment by Kintonig** [comment]
Hello and thank you for the app. My gis and loves it and I was wondering, if you are going to create an IOS’ version. 
‘or if I try to compilate myself, I’m not sure what it would do 
Thanks for your answers

**Reply by fast4x** [reply]
Hi, RiMusic repository it's ready to develop iOS version if any developers is interested. If you want you can try, but i recommend you careful to avoid bugs in other versions. The project is cross-platform, currently contains the android version and desktop versions in development.
